packages/kbn-test/src/functional_test_runner/cli.ts

```diff
@@ -48,6 +48,10 @@ export function runFtrCli() {
           kbnTestServer: {
             installDir: parseInstallDir(flags),
           },
+          suiteFiles: {
+            include: toArray(flags['include-file'] as string | string[]),
+            exclude: toArray(flags['exclude-file'] as string | string[]),
```

Comment:
I think we should resolve these paths to absolute paths here so that relative paths can be passed on the CLI relative to the current working directory, as seems to be customary.